extensions/snowflake/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "snowflake"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+
+[lib]
+crate-type = ["cdylib"]
+
+[profile.release]
+opt-level = "z"
+strip = true
+lto = true
+codegen-units = 1
+
+[dependencies]
+grafbase-sdk = { path = "../../crates/grafbase-sdk" }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+snowflake-api = "0.6"
+tokio = { version = "1", features = ["rt"] }
+deadpool = "0.9"
```

Comment:
Why do we need deadpool here? The snowflake client is using http, which has pooling already in every implementation.